frontend/packages/metal3-plugin/src/selectors/secret.ts

```diff
@@ -0,0 +1,6 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
```

Comment:
This file should go in `@console/shared` but I am fine with doing it in a follow up.